src/NServiceBus.Transport.Msmq/Persistence/SubscriptionStorage/MsmqSubscriptionPersistence.cs

```diff
@@ -9,15 +11,24 @@ class MsmqSubscriptionPersistence : Feature
     {
         protected override void Setup(FeatureConfigurationContext context)
         {
-            var queueName = context.Settings.GetConfiguredMsmqPersistenceSubscriptionQueue();
+            var configuredQueueName = context.Settings.GetConfiguredMsmqPersistenceSubscriptionQueue();
```

Comment:
Does the name `configuredQueueName` is an improvement over `queueName`?

Reply:
@ramonsmits - Yes, I renamed it so that its clear from the variable name that's the value that was configured by the user and not the default value. The code I felt was more readable after I made these simple changes.